### ISSUE TITLE
[Feature Request] v-data-table : Customize the v-checkbox header in the headers array 

### ISSUE THREAD

**Issue by Kleemer** [vuetifyjs/vuetify#3486]
**New Functionality**
Allow to customize only some headers in a Data table. 

I want to add some custom classes to the checkbox column, but not to my other columns. I know I can add some classes in the headers array, but there is no way to access the checkbox header other than using `<template slot="headers" slot-scope="props">`, but if I do that I have to manually iterate in my headers array instead of having the default behavior.

**Improvements**
The only workaround is to customize all headers using the "headers" slot, but that is not needed in my case and I guess I don't see why we cannot easily access the v-checkbox header.
 
**Bugs or Edge Cases it Helps Avoid**
More friendly use of the v-checkbox header.

**Comment by nekosaur** [comment]
Please show how would you imagine the final markup of this feature would look like

**Comment by Kleemer** [comment]
I guess I would see something like that :

```[...]
data () {
      return {
        headers: [
          { type: 'checkbox', class: "checkbox-class" }
          { sortable: false },
          { text: 'Title', value: 'title' },
          { text: 'Something', value: 'smth' },
          { text: 'Other', value: 'other' }
        ]
      }
    }
[...]
```
We should be able to describe the first header column as 'checkbox', and then give the same properties as the other headers.

The only other possibility would be a new props 'select-all-class' however I do find that a little bit less elegant.

**Comment by blalan05** [comment]
available in the 2.0 alpha.

**Comment by ManishaDhingraNiit** [comment]
is there any work around for this feature for release "vuetify": "^1.5.1",
